srcpkgs/boost/template

```diff
@@ -1,36 +1,37 @@
 # Template file for 'boost'
 pkgname=boost
-version=1.72.0
+version=1.73.0
 revision=1
 wrksrc="${pkgname}_${version//\./_}"
 hostmakedepends="which bzip2-devel icu-devel python-devel python3-devel"
-makedepends="zlib-devel bzip2-devel icu-devel python-devel python3-devel"
+makedepends="zlib-devel bzip2-devel icu-devel python-devel python3-devel
+ libzstd-devel liblzma-devel"
 short_desc="Free peer-reviewed portable C++ source libraries"
 maintainer="John <johnz@posteo.net>"
 license="BSL-1.0"
 homepage="http://www.boost.org/"
 distfiles="https://dl.bintray.com/boostorg/release/${version}/source/boost_${version//./_}.tar.bz2"
-checksum=59c9b274bc451cf91a9ba1dd2c7fdcaf5d60b1b3aa83f2c9fa143417cc660722
+checksum=4eb3b8d442b426dc35346235c8733b5ae35ba431690e38c6a8263dce9fcbb402
 
 CXXFLAGS="-std=c++14"
 
-subpackages="libboost_program_options1.72 libboost_math_c99f1.72
```

Comment:
This large diff makes me think about replacing all of 1.73 with:

     ${version%.*}

And hope this list will be stable enough.

Reply:
Sure might be possible, but doing it only in subpackages would not help much. And I don't want to go the libreoffice template route of having a function to generate all sub package functions.

Reply:
Yes, and this is just a `s/1[.]72/1[.]73/g`, hence just a general comment from me.